src/Subsystems/Intake.cpp

```diff
@@ -17,11 +17,13 @@ std::shared_ptr<Intake> Intake::getInstance() {
 }
 
 Intake::Intake() : Subsystem(kSubsystemName),
-    intakeMotor(RobotMap::kIDIntakeMotor),
+    intakeMotorTop(RobotMap::kIDIntakeMotorTop),
+    intakeMotorBottom(RobotMap::kIDIntakeMotorBottom),
     intakeSolenoid(RobotMap::kIDIntakeForward, RobotMap::kIDIntakeReverse)
 {
     intakeSolenoid.Set(DoubleSolenoid::kForward);
-    intakeMotor.SetInverted(true);
+    intakeMotorTop.SetInverted(true);
+    intakeMotorBottom.SetInverted(false);
```

Comment:
This doesn't work because the climber subsystem is putting it back.

Reply:
I'll bet the bottom motor can also be made a follower. (Unless we want different speeds.)

No change necessary.